.github/actions/generate-random-port/action.yml

```diff
@@ -0,0 +1,12 @@
+name: 'generate_port'
+description: 'generate_port'
+outputs:
+  random-port:
+    description: "Random port"
+    value: ${{ steps.random-port-generator.outputs.random-port }}
+runs:
+  using: "composite"
+  steps:
+  - id: random-port-generator
+    run: echo "::set-output name=random-port::$(bash ${{ github.action_path }}/generate_port.sh)"
+    shell: bash
```

Comment:
Since verdictd and inclavare-containers are in the same organization. Could you use the github/action in inclavare-containers/inclavare-containers repo.

Reply:
I think it's too complicated to down this file in inclavare-containers repo if user only want to build verdictd repo.

Reply:
Investigated and can't find a simple way to use the action as a shared action. Will open a issue.